phpBB/phpbb/console/application.php

```diff
@@ -35,7 +35,7 @@ class application extends \Symfony\Component\Console\Application
 	* @param string			$version	The version of the application
 	* @param \phpbb\user	$user		The user which runs the application (used for translation)
 	*/
-	public function __construct($name, $version, \phpbb\user $user)
+	public function __construct($name, $version, $user)
```

Comment:
probably because we don't have the user object there, but that was done by @Nicofuma